packages/lambdas/src/ihe.ts

```diff
@@ -1,9 +1,9 @@
 import * as Sentry from "@sentry/serverless";
 import { capture } from "./shared/capture";
 import { APIGatewayProxyEvent } from "aws-lambda";
-import { generateITI38 } from "@metriport/core/external/carequality/iti-38-parsing";
-import { generateITI39 } from "@metriport/core/external/carequality/iti-39-parsing";
-import { generateXCPD } from "@metriport/core/external/carequality/iti-55-parsing";
+import { generateITI38 } from "@metriport/core/external/carequality/dq/dq-parsing";
+import { generateITI39 } from "@metriport/core/external/carequality/dr/dr-parsing";
+import { generateXCPD } from "@metriport/core/external/carequality/xcpd/xcpd-parsing";
```

Comment:
nit: why `xc` on `xcpd`? wouldn't just `pd` better match the pattern of `dq` and `dr`? Like, we don't use `xca` on the other ones as its part of the protocol

Reply:
good point. I was just using the business terms I had heard everyone using and xcpd seemed to be what we were saying instead of iti-55. I don't even know what xc stands for!